org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterIlluminance.java

```diff
@@ -86,7 +89,8 @@ public void attributeUpdated(ZclAttribute attribute) {
                 && attribute.getId() == ZclIlluminanceMeasurementCluster.ATTR_MEASUREDVALUE) {
             Integer value = (Integer) attribute.getLastValue();
             if (value != null) {
-                updateChannelState(new DecimalType(BigDecimal.valueOf(value, 2)));
+                updateChannelState(new QuantityType<Illuminance>(BigDecimal.valueOf(value, 2),
```

Comment:
I had a look in to the cluster library spec (4.2.2.1.1), and it states that the measured value is *not* the illuminance in Lux, but rather `measuredValue = 10.000 x log10(Illuminance) + 1`. Hence, quantifying the measured value in Lux is probably not correct here.

Reply:
Looked at the spec too and agree that apparently the measured value is not in Lux. Will revoke this change and restore the previous version.